client/app/telecom/pack/xdsl/modem/templates/config/pack-xdsl-modem-templates-config.html

```diff
@@ -0,0 +1,246 @@
+        </div> <!-- End of Error Display -->
+
+        <!-- Templates list display -->
+        <div data-ng-if="$ctrl.templates.length > 1 && !$ctrl.template">
+            <div class="widget-presentation">
+                <header class="widget-presentation-header">
+                    <h2 class="widget-presentation-title"
+                        data-translate="xdsl_modem_template_config_list_title">
+                    </h2>
+                </header>
+            </div>
+
+            <oui-datagrid data-rows="$ctrl.templates" data-page-size="10">
+                <oui-column data-title="'xdsl_modem_template_config_table_name' | translate"
+                            data-property="name" data-sortable="asc">
+                    {{$row.name}}
+                </oui-column>
+                <oui-column data-title="'xdsl_modem_template_config_creation_date' | translate"
+                            data-property="detail.creationDate" data-sortable>
+                    <span data-ng-if="$row.detail.creationDate" data-ng-bind="$row.detail.creationDate | date:'dd/MM/yyyy HH:mm:ss'">
+                    </span>
+                    <span data-ng-if="!$row.detail.creationDate" data-ng-bind="$row.detail.info">
+                    </span>
+                </oui-column>
+
+                <oui-action-menu data-align="end" data-compact>
+                    <oui-action-menu-item on-click="$ctrl.setSelectedTemplate($row)">
+                        <span data-translate="edit"></span>
+                    </oui-action-menu-item>
+                    <oui-action-menu-item on-click="$ctrl.deleteSelectedTemplate($row)">
+                        <span data-translate="delete"></span>
+                    </oui-action-menu-item>
+                </oui-action-menu>
+
+            </oui-datagrid>
+
+        </div> <!-- End of Templates list display -->
+
+        <div data-ng-if="$ctrl.template"> <!-- Template Configuration -->
+            <div class="widget-presentation">
+                <header class="widget-presentation-header">
+                    <h2 class="widget-presentation-title"
+                        data-translate="xdsl_modem_template_config_title"
+                        data-translate-values="{ name: $ctrl.template.name }">
+                    </h2>
+                </header>
+            </div>
+            <form class="mb-3"
+                    name="templateConfigForm"
+                    data-ng-submit="$ctrl.updateTemplate()"
+                    novalidate>
+
+                <!-- Empty config Display -->
+                <oui-message data-type="warning" data-ng-if="$ctrl.template.error">
+                    {{$ctrl.template.detail.capabilities}}
+                </oui-message><!-- End of Empty config Display -->
+
+                <!-- DHCP configuration -->
+                <div class="form-group">
+                    <h5 data-translate="xdsl_modem_template_config_dhcp"></h5>
```

Comment:
You jumped from `h2` to `h5`, please use `h3` instead

Reply:
the style h3 is bigger than h5, than I wanted. I'll do this if you prefer

Reply:
You can use `.oui-heading_5` to get the style you want :)